src/lib/pubkey/curve448/x448/curve448.cpp

```diff
@@ -0,0 +1,129 @@
+/*
+* Curve448
+* (C) 2024 Jack Lloyd
+*     2024 Fabian Albert - Rohde & Schwarz Cybersecurity
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include <botan/curve448.h>
```

Comment:
As mentioned in the earlier comment, just s/Curve/X/ everywhere when referring to the key exchange algorithm.